.gitignore

```
@@ -129,3 +129,4 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+.idea
```

This should get rid of the .idea/.gitignore file
```suggestion
.idea/
```